src/components/waste/WasteCollectionListItem.tsx

```diff
@@ -37,7 +37,17 @@ export const WasteCollectionListItem = ({ item: groupedItem, options }) => {
                           {usedTypes[typeKey].color !== usedTypes[typeKey].selected_color && (
                             <Dot color={usedTypes[typeKey].selected_color} />
                           )}
-                          <RegularText> {usedTypes[typeKey].label}</RegularText>
+                          <RegularText> {usedTypes[typeKey].label} </RegularText>
+                          {item.note && (
+                            <Tooltip
+                              backgroundColor={colors.darkText}
+                              containerStyle={{ height: 'auto' }}
+                              popover={<RegularText lightest>{item.note}</RegularText>}
+                              withPointer
+                            >
```

Comment:
i suggest adding these props, that we use for waste collection settings tooltip already:

```
withOverlay={device.platform === 'android'}
overlayColor={colors.shadowRgba}
```

on iOS having the overlay above the content is not a common practice.

Reply:
did you change that? do you have new screenshots to check it out for iOS and Android please?

Reply:
|before ios|after ios|before android|after android|
|--|--|--|--|
![Simulator Screenshot - iPhone 16 Pro Max - 2025-04-08 at 08 03 09](https://github.com/user-attachments/assets/288ada34-690d-47fb-8863-50159a9763c8)|![Simulator Screenshot - iPhone 16 Pro Max - 2025-04-08 at 08 03 40](https://github.com/user-attachments/assets/b79c8afc-7745-46f0-9d7b-2beca1029d7f)|<img width="323" alt="SCR-20250408-hpel" src="https://github.com/user-attachments/assets/3b0ea26d-5d15-4828-ab67-56c3abcc4fb7" />|<img width="319" alt="SCR-20250408-hoyo" src="https://github.com/user-attachments/assets/7f2e32e5-c000-4788-8724-a466b1070e1d" />